ci/make_container_images.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# This script is intended to be used by Cirrus-CI, from the VM
+# built by the 'image_builder' makefile target in this repo.  Use
+# of this script in any other context/environment is unlikely to
+# function as intended.
+
+set -e
+
+SCRIPT_FILEPATH=$(realpath "$0")
+SCRIPT_DIRPATH=$(dirname "$SCRIPT_FILEPATH")
+REPO_DIRPATH=$(realpath "$SCRIPT_DIRPATH/../")
+
+# shellcheck source=./lib.sh
+source "$REPO_DIRPATH/lib.sh"
+
+# When running under Cirrus-CI on a branch, also tag/push "latest" images
+PUSH_LATEST=0
+
+# shellcheck disable=SC2154
+if [[ -z "$CI" ]] || [[ "$CI" != "true" ]] || [[ "$CIRRUS_CI" != "$CI" ]]; then
+    die "Unexpected \$CI=$CI and/or \$CIRRUS_CI=$CIRRUS_CI"
+elif [[ -z "$IMG_SFX" ]]; then
+    die "Required non-empty values for \$IMG_SFX=$IMG_SFX"
```

Comment:
Can you make these error messages clearer to the reader? "Required ..." does not give me a good indication of what the problem is. Perhaps simply "Required envariable \$IMG_SFX is undefined". Likewise for the next two.

Reply:
Oh one missing bit of context here is: the `die()`  function will prefix the message with `ERROR:`.  Does that sway your opinion?

Reply:
No, my concern is that "Required" is ambiguous in this context: it isn't clear if it's an adjective or a past-tense verb. Someone reading that message (like me) might interpret it as "The required non-empty values for \$IMG_SFX ... [then what? incomplete sentence?]". (Also, it doesn't seem very helpful to add `=$VAR` when you've just confirmed that `$VAR` is empty). If you like the word, may I suggest "I require non-empty values for..." or "this tool requires non-empty values for..."?